### ISSUE TITLE
Deep Learning for Health Informatics

### ISSUE THREAD

**Issue by ghuijo** [yskim0/One-Day-One-Paper#137]
## Deep Learning for Medical Imaging
* Computer-aided diagnosis (CAD) can provide an objective assessment of the underlying disease processes
* CNN을 많이 활용함
  * outstanding performance demonstrated in computer vision
  * ability to be parallelized with GPUs
* CAD의 챌린지들
  * differences in shape and intensity of tumors/lesions
  * the variations in imaging protocol
  * MRI cannot be handled automatically using simpler machine learning approaches
* hybrid approaches that combine CNNs with other architectures
  * hybridCNNwith extreme learning machine (ELM)
* CNN 주요 챌린지 3가지
  * CNNs are designed for 2-D images whereas segmentation problems in MRI and CT are inherently 3-D
  * CNNs do not model spatial dependencies
  * Preprocessing to bring all subjects and imaging modalities to similar distribution is still a crucial step
  * 위에는 작고 incomplete한 training datasets으로 인한 문제들임.
  * crowdsourcing & transfer learning via fine tuning 제안됨.
* stacked denoising autoencoder
  * outperforms classical CAD approaches
  * automatic feature extraction과 noise tolerance가 성능 향상의 요인.
  * image segmentation 과정을 없앰.
* deep Autoencoder combined with a softmax output layer for regression
* low level image preocessing에서는 manifold learning approach of 3-D brain images based on DBN
* special NN with polynomial activation function과 principal component-regression analysis 함께 사용.

## Medical Informatics
* 기존 EHR 연구의 dataset 한계
  * The quality of reporting varies considerably among institutions and persons
  * Multimodal datasets of several petabytes
  * Long-term time dependencies between clinical events and disease diagnosis and treatment that complicates learning
  * Inability of traditional machine learning approaches to scale up to large and unstructured datasets
  * Lack of interpretability of results hinders adaptation of the methods in the clinical setting
* 그래서 딥러닝의 활용이 중요함.
* a stack of denoising autoencoders can be used to automatically infer features from a large-scale EHR database
* DNNs have significantly higher prediction accuracies than conventional approaches

##
* 거의 문장 단위로 논문&기술이 쏟아짐.
* health informatics는 생각보다 넓은 영역이라서 나중에 참고할 만한 연구 위주로 정리함.
